source/walkthroughs/deploy/install_language_runtime/_ruby.md.erb

```diff
@@ -120,9 +120,9 @@ sudo ln -sf /usr/bin/nodejs /usr/local/bin/node</pre>
       <p>Run the following commands to install Node.js from the <a href="https://nodesource.com/">NodeSource</a> APT repository.</p>
       <pre class="highlight">sudo apt-get update &amp;&amp;
 sudo apt-get install -y apt-transport-https ca-certificates &amp;&amp;
-curl --fail -ssL -o setup-nodejs https://deb.nodesource.com/setup_0.12 &amp;&amp;
+curl --fail -ssL -o setup-nodejs https://deb.nodesource.com/setup_8 &amp;&amp;
```

Comment:
This command is incorrect (missing .x?)

Reply:
I think we should also merge this with the Debian >= 8 (Jessie or later) row above, because Debian moves pretty slow compared to Node (Debian 8 is still on 0.10, and Debian 9 is on 4.8 which will be EOL in a few months). It's probably best to just use the nodesource APT instructions.

Reply:
This one I was not actually very sure about because it is relevant to general passenger installation, not necessary a node/meteor constraint. I'd still say any node installation instruction should simply reflect the latest LTS available (8.x), but this would be your call. What do you think?

Reply:
IMHO we should place a link to [node.js installation instructions](https://nodejs.org/en/download/), but Meteor requirement is `v8.8.1`, while latests LTS is `v8.9.1`, not sure what we can do about this. Tell to the user check *recommended* node version via?:
```shell
meteor node -v
```

Reply:
My main concern is that we don't advise installing stuff that is EOL, and from that perspective the instructions need to be changed for Debian 7 / 8 / 9 anyway.

For maintainability it's easiest to point to the APT method (since those instructions don't change like the nodesource link) and I don't mind pointing to old LTS instructions as long as they're not EOL soon. But specifically for Debian it looks like that's not really feasible as long as Node is moving as fast as it does.

Reply:
@OnixGH if it wouldn't be too complex I may provide instructions (*or a bash script*) for downloading and compiling needed version of Node (*which will work regardless to Linux version and distribution*).

Reply:
Btw if we're looking for simpler solution NVM will work too.